docs/content/manual/rancher-integration/drain-using-rancher-ui.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Drain using Rancher UI
+---
+
+Test with Longhorn default setting of 'Node Drain Policy': `block-if-contains-last-replica`
+----
+
+### 1. Drain operation on single node using Rancher UI
+
+**Given**  Single node cluster with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND 1 RWO and 1 RWX volumes unattached
+
+**When** Drain the node with default values of Rancher UI
+
+**Then** Drain should be blocked
```

Comment:
Regarding the test step `Drain should be blocked`, I have observed that the node status can change from `Active` to `Draining` and back to `Cordoned` in a loop. Is this the expected behavior?

Reply:
I imported k3s into Rancher v2.7.1 in below scenario:

- 1 node (control-plane), the drain button hideen hence I can not execute drain operation
- 2 nodes (1 control and 1 worker) which controlplane tainted NoSchedule, LH only deploy on worker node, when drain the worker node, I also faced draining -> cordoned loop, because no other available node for pod can move on, so the drain not finish I think, should we consider this scenario into test steps?